src/paraswap.ts

```diff
@@ -1,8 +1,14 @@
 import axios, { AxiosError, AxiosRequestConfig } from 'axios';
 import * as qs from 'qs';
 import * as _ from 'lodash';
```

Comment:
would be nice to also remove that import and directly import the lodash chunks you need `import isEmpty from 'lodash/isEmpty'`